sdk/communication/azure-communication-sms/README.md

```diff
@@ -21,7 +21,7 @@ Azure Communication SMS is used to send simple text messages.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-communication-sms</artifactId>
-  <version>1.0.0-beta.4</version>
+  <version>1.0.0-beta.5</version>
```

Comment:
Is this supposed to be beta.5 or 6? I see that in the pom.xml below, it is 6 but here, its 5

Reply:
SMS and phonenumbers are going to be on beta.5 .. I think I should have added this in the notes